SU2_CFD/include/drivers/CDiscAdjSinglezoneDriver.hpp

```diff
@@ -38,7 +42,12 @@
  */
 class CDiscAdjSinglezoneDriver : public CSinglezoneDriver {
 protected:
-
+#ifdef CODI_FORWARD_TYPE
+    using Scalar = su2double;
+#else
+    using Scalar = passivedouble;
```

Comment:
You may need this to be consistent with CSolver
```suggestion
    using Scalar = su2mixedfloat;
```

Reply:
Thank you!